content/en/agent/troubleshooting/high_memory_usage.md

```diff
@@ -0,0 +1,64 @@
+---
+title: High CPU or Memory Consumption
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "/agent/troubleshooting/send_a_flare/"
+  tag: "Agent Troubleshooting"
+  text: "Send an Agent Flare"
+- link: "/agent/troubleshooting/agent_check_status/"
+  tag: "Agent Troubleshooting"
+  text: "Get the Status of an Agent Check"
+---
+
+Several factors can cause high Agent CPU or memory consumption. If you try the steps below and continue to have trouble, [contact Datadog Support for further assistance](#reach-out-to-datadog-support).
+
+Common causes of high CPU or memory consumption:
```

Comment:
should we make this an H3 or so? might make the page easier to parse at-a-glance
```suggestion
### Common causes of high CPU or memory consumption:
```